oauth-function-template/datastore/table.js

```diff
@@ -12,6 +12,7 @@ const _createOAuthTableIfNotExists = (ctx) => {
   return ctx.graphql.query(queries.findTableQuery)
       .then((result) => {
         if (result.errors && result.data.node.table === null) {
+          console.log('Creating OAuth table as it does not exist');
```

Comment:
How about additional log after L18 with message about the fact that table was created successfully ? IMO, it would be good to see such a log when debugging issues related to oauth flow.
\+ maybe we can consider also adding log after each successful graphql.querry request?

Reply:
How about log after L72  saying that we are creating ouath table columns ?

Reply:
I'll add graphql requests logging in a separate PR